CONTRIBUTING.md

```diff
@@ -110,6 +110,11 @@ parsed from a string (during a network capture or a PCAP file
 read). Adding inefficient code here will have a disastrous effect on
 Scapy's performances.
 
+### Python 2 and 3 compatibility
+
+The project aims to provide code that works both on Python 2 and Python 3. Therefore, some rules need to be apply to achieve compatibility:
+- byte-string must be defined as b"\x00\x01\x02"
+
```

Comment:
Could you also add a line related to catching exceptions, and writing lambdas with more than one argument ?